.dockerignore

```diff
@@ -0,0 +1,5 @@
+.github
+config
+modules
+build.sh
+test-build.sh
```

Comment:
what about the .git directly, should it also be ignored?

Reply:
Please see my comment https://github.com/bastilimbach/docker-MagicMirror/pull/46#issuecomment-655339092